examples/tv-app/linux/include/content-launcher/ContentLauncherManager.h

```diff
@@ -30,10 +30,17 @@
 class ContentLauncherManager : public chip::app::Clusters::ContentLauncher::Delegate
 {
 public:
+    ContentLauncherManager() : ContentLauncherManager({ "example", "example" }, 0){};
+    ContentLauncherManager(std::list<std::string> acceptHeaderList, uint32_t supportedStreamingProtocols);
+
     LaunchResponse HandleLaunchContent(chip::EndpointId endpointId, const std::list<Parameter> & parameterList, bool autoplay,
                                        const chip::CharSpan & data) override;
     LaunchResponse HandleLaunchUrl(const chip::CharSpan & contentUrl, const chip::CharSpan & displayString,
```

Comment:
an endpointId is for HandleLaunchContent but not f or HandleLaunchUrl?

Reply:
Since a delegate can now be assigned to a specific endpoint, I don't believe that we need endpoint to be passed to delegate methods. This is left over from the previous convention and can be removed.

Reply:
Removed the endpoint argument for now to make things consistent